### PR TITLE
ConvertToDecimalInt method implemented

### PR DESCRIPTION
**Pull request by LimbertVargas**
I have just finished implementing the code, please review it.
Regards

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (J0c4) [inline .classpath]: Do we really need to version this file? if not, let's add it to `.gitignore`

File: .project [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (J0c4) [inline .project]: Do we really need to version this file? if not, let's add it to `.gitignore`

File: src/main/java/binaries/limbert/BinaryArray.java [hunk 3]
```diff
@@ -18,6 +18,12 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        //throw new UnsupportedOperationException("Not supported yet.");
```
Comment (J0c4) [inline src/main/java/binaries/limbert/BinaryArray.java]: Please remove commented line

File: src/main/java/binaries/limbert/BinaryArray.java [hunk 4]
```diff
@@ -18,6 +18,12 @@ public BinaryArray(int[] binaries) {
...
+    	int longitud = binaries.length;
```
Comment (J0c4) [inline src/main/java/binaries/limbert/BinaryArray.java]: Let's use english as standard for names in the code

File: src/main/java/binaries/limbert/BinaryArray.java [hunk 5]
```diff
@@ -18,6 +18,12 @@ public BinaryArray(int[] binaries) {
...
+		int dec = 0;
```
Comment (J0c4) [inline src/main/java/binaries/limbert/BinaryArray.java]: be aware of the abbreviated names, sometimes they are not as meaningful as their non-abbreviated version

File: src/main/java/binaries/limbert/BinaryArray.java [hunk 6]
```diff
@@ -18,6 +18,12 @@ public BinaryArray(int[] binaries) {
...
+		for (int i = longitud-1; i > 0; i--) {
+			dec = dec+binaries[i] * (int) Math.pow(2, longitud-1 - i);
```
Comment (J0c4) [inline src/main/java/binaries/limbert/BinaryArray.java]: spaces missing before and after `+` `-` operators. Plus `2` like magic number

**Review by J0c4 — CHANGES_REQUESTED** [review]
Good, solution works as expected, need to address my comments in the PR

**Comment by J0c4** [comment]
Hello @LimbertVargas - You latests updates look good, however, the `.classpath` and `.project` still need to be deleted. Did you run `git rm .classpath` and `git rm .project`? those should remove the files

**Comment by LimbertVargas** [comment]
Hello Jose,
I will delete it, Thank you

Regards,

[Fundacion Jala]
Limbert Alvaro Vargas Laura
Becario - Fundacion Jala
________________________________

**Pull request merged**